packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -122,18 +122,18 @@ const AutocompleteRoot = experimentalStyled(
       width: 0,
       minWidth: 30,
     },
-    '&.MuiInput-root': {
+    '& .MuiInput-root': {
```

Comment:
I don't think this changes [selector specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). You only changed the combinator and these shouldn't have any effect.